Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 Language Features:
  * General: Support ``pop()`` for storage arrays.
+Compiler features:
+ * Parser: Introduce ``emit`` as a keyword instead of parsing it as identifier.
```

Comment:
I think this is actually a breaking change, because `emit` cannot be used as an identifier anymore. Moved.